src/libraries/System.Private.CoreLib/src/System/TimeSpan.cs

```diff
@@ -351,7 +352,7 @@ public static bool TryParse(ReadOnlySpan<char> input, IFormatProvider? formatPro
         {
             return TimeSpanParse.TryParse(input, formatProvider, out result);
         }
-        public static bool TryParseExact(string? input, string format, IFormatProvider? formatProvider, out TimeSpan result)
+        public static bool TryParseExact([NotNullWhen(true)] string? input, string format, IFormatProvider? formatProvider, out TimeSpan result)
```

Comment:
We're missing `?` on `format`.  Mind fixing that while you're at it?  It could also be `[NotNullWhen(true)]`.